README.md

```diff
@@ -17,20 +19,20 @@ development-related projects are now becoming larger and more complex and they n
 intermediate software engineering skills to help them design more robust software code, 
 automate the process of testing and verifying its correctness and support collaborations with others.
 
-## About the Lesson
-
-This lesson teaches intermediate-level software development skills in a way that mimics a typical software development 
-process in a team, starting from an existing piece of software.
-
-The software project template repository used for this lesson is available at:
-<https://github.com/carpentries-incubator/python-intermediate-inflammation>.
-
 ### Lesson Status
-The lesson is currently in beta - it has been tested several times with 
-different cohorts by the lesson authors as well as independently by people not involved in the course development
+
+The lesson is currently in beta - it has been over 10 times times with 
+different cohorts by the lesson authors as well as independently by people not directly involved in the lesson development
 and is in a good state to be reused and taught by others.
 
 ## Teaching the Lesson
+
+The lesson has 5 sections that can be delivered (approximately) over 5 half-days. The lesson is suitable either for 
```

Comment:
Should we say 5 half-days, or 5 two-third days? (as I think some of the sections do take longer than a hlaf-day).

Reply:
Yes, I was wondering how to indicate the time needed most accurately. Perhaps we could just say 5 days?

Reply:
I'm still not 100% certain on this. There's some unevenness in the length of time needed for each of the 5 sections take to teach. When teaching in person we found that we over ran for several of the earlier, coding, lessons. But regained time when teaching the later theoretical lessons.

Perhaps play safe, and suggest that teachers put aside 5 days for the course. But state that each section will take between half a day, and a full day, to teach.